pcsx2/Config.h

```diff
@@ -1228,7 +1228,7 @@ struct Pcsx2Config
 		// TODO - Vaser - where are these settings exposed in the Qt UI?
 		EnableRecordingTools : 1,
 		EnableGameFixes : 1, // enables automatic game fixes
-		SaveStateOnShutdown : 1, // default value for saving state on shutdown
+		SaveStateOnShutdown : 0, // default value for saving state on shutdown
```

Comment:
This is the number of bits, not the default value. The default value for the bitsets is in Pcsx2Config.cpp, and if it's not mentioned explicitly, it's defaulted to 0/false.

Reply:
thanks, reverted it right now